src/SpecialExamWrapper/Sequence.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { AppContext } from '@edx/frontend-platform/react';
+import { Spinner } from '@edx/paragon';
+import { ExamInstructions } from '../ExamInstructions';
+import {
+  getExamData,
+  getAttemptData,
+  startExam,
+  store,
+} from './data';
+
+const mapCoursewareStateToProps = (state) => {
+  const { courseware } = state;
+  return { courseId: courseware.courseId };
```

Comment:
Why do we need additional const { courseware } if we can return { courseId: state.course.Id} or I've missed something?